osquery/utils/conversions/windows/windows_time.cpp

```diff
@@ -27,4 +29,38 @@ LONGLONG longIntToUnixtime(LARGE_INTEGER& li) {
   return ull.QuadPart / 10000000ULL - 11644473600ULL;
 }
 
+// Convert little endian Windows FILETIME to unix timestamp
+LONGLONG littleEndianToUnixTime(const std::string& time_data) {
+  // If timestamp is zero dont convert to UNIX Time
+  if (time_data == "0000000000000000") {
+    return 0LL;
+  } else {
+    std::string time_string = time_data;
+    // swap endianess
+    std::reverse(time_string.begin(), time_string.end());
```

Comment:
Is this comment correct?

Reply:
It reverses the string and then uses a the for loop below to to swap each char of the string
It converts the little endian string to a big endian string so it can be converted from FILETIME to unix time

Reply:
I took another look at this and changed the for loop logic to use `std::swap`